python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_magentic_one/_magentic_one_orchestrator.py

```diff
@@ -427,7 +428,7 @@ def _thread_to_context(self) -> List[LLMMessage]:
         """Convert the message thread to a context for the model."""
         context: List[LLMMessage] = []
         for m in self._message_thread:
-            if isinstance(m, ToolCallRequestEvent | ToolCallExecutionEvent):
+            if isinstance(m, ToolCallRequestEvent | ToolCallExecutionEvent | ToolCallSummaryMessage):
```

Comment:
Actually we should not ignore tool call summary messages -- they should be handled just like text message etc.

Reply:
Fixed.  I took my lead from the `#ignore tool call messages` comment.   Addressing now.